docs/reference/commands.rst

```diff
@@ -73,7 +73,11 @@ upgrade phase. Refer to the output below for a description of all available opti
                             Default to enabling database backup.
       --force               Force the plan/upgrade of non-empty hypervisors.
       --verbose, -v         Increase logging verbosity in STDOUT.
-                            Multiple 'v's yield progressively more detail (up to 3).
+                            Multiple 'v's yield progressively more detail (
+                            
+                            
+                            
+                            3).
```

Comment:
Why this in on 5 lines?

Reply:
Seems to be a merging hiccup. Fixed, thanks